test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -3946,10 +3946,6 @@ def log_lines(self, logtype, id=None, n=50, tail=True, day=None,
                     else:
                         lines = self.du.cat(self.hostname, filename, sudo=sudo,
                                             level=logging.DEBUG2)['out']
-                # tail is not a standard, e.g. on Solaris it does not recognize
-                # -n. We circumvent this problem by using PTL's version of tail
-                # but it currently only works on a local host, for remote hosts
-                # we fall back to using tail/head -n
                 elif self._is_local and not sudo:
                     if tail:
                         futils = FileUtils(filename, FILE_TAIL)
```

Comment:
Just FYI: FileUtils was introduced only for platform when tail/head doesn't understand -n argument. Since now all supported platform understand we should remove FileUtils also. But off-course not in this PR.

Reply:
Yes sure. Later perhaps?